src/aleph/http/client.clj

```diff
@@ -453,9 +482,12 @@
      decompress-body? false
      save-content-encoding? false}}]
   (fn [^ChannelPipeline pipeline]
-    (let [handler (if raw-stream?
-                    (raw-client-handler response-stream response-buffer-size)
-                    (client-handler response-stream response-buffer-size))
+    (let [error-logger' (or error-logger
+                            (fn [^Throwable ex]
+                              (log/warn "error in HTTP client" (.getMessage ex))))
```

Comment:
Do you ignore stacktrace for the exception on purpose? What I mean is that `(log/warn ex "error in HTTP client")` would print both the exception message and its trace, but your code will only print the message itself. It's easy to get it by using the custom logger, but I believe having traces by default will actually be useful for more people.